Assets/MRTK/Services/InputSimulation/InputSimulationService.cs

```diff
@@ -340,7 +345,7 @@ public override void LateUpdate()
                     UpdateHandDevice(HandSimulationMode, Handedness.Right, HandDataRight);
 
                     // HandDataGaze is only enabled if the user is simulating via mouse and keyboard
-                    if (UserInputEnabled)
+                    if (UserInputEnabled && SimulateGaze && !(HandDataLeft.IsTracked || HandDataRight.IsTracked))
```

Comment:
There's also an `IsInteractionEnabled` on the gaze pointer that might be usable here. It's managed by the gaze pointer visibility state machine and should behave a deterministic way across platforms. Can we replace `SimulateGaze` with that?

Reply:
My concern with making hard changes to the gaze pointer is that it might modify old HoloLens 1 behavior (since making changes to the gaze pointer seems to get surfaced when I use the gestures input simulation mode)

Reply:
I don't think we'd need to make any changes to the gaze pointer, but instead only simulate gaze here if `CoreServices.InputSystem.GazeProvider.GazePointer.IsInteractionEnabled` is `true`. I _think_ this case should be handled by that property already

Reply:
Might be interesting to see how using that value with `isSimulatingGaze` in `SimulatedHandDataProvider` affects things too.